pkg/middleware/rate_limit.go

```diff
@@ -0,0 +1,19 @@
+package middleware
+
+import (
+	"golang.org/x/time/rate"
+	"gopkg.in/macaron.v1"
+
+	"github.com/grafana/grafana/pkg/models"
+)
+
+// A very basic rate limiter. Will alow average of "rps" requests per second over an extended period of time, with max "burst" requests at the same time
```

Comment:
Go doc comments should be a sentence starting with what's being described.
```suggestion
// RateLimit is a very basic rate limiter. Will allow average of "rps" requests per second over an extended period of time, with max "burst" requests at the same time.
```